client/src/main/java/io/spine/client/Filters.java

```diff
@@ -565,6 +562,27 @@ public static CompositeFilter either(Filter first, Filter... rest) {
         return composeFilters(asList(first, rest), EITHER);
```

Comment:
A question, similar to methods accepting `Column<?, ?>`. This part of code was not modified by this PR, but the methods accepting `EntityStateField` also accept `Object` as the second parameter.

Can we have `EntityStateField` and `SubscribableField` extended (in one of the following PRs in `base`) so that they have a generic parameter with the type of the referenced field? It would allow to make query- and subscription- API more type-safe.